frb_rust/src/support.rs

```diff
@@ -42,11 +41,37 @@ pub unsafe fn box_from_leak_ptr<T>(ptr: *mut T) -> Box<T> {
     Box::from_raw(ptr)
 }
 
+mod sealed {
+    pub trait Sealed {}
+    impl Sealed for i64 {}
+    impl Sealed for u64 {}
+}
+
+/// Cast a byte buffer into a boxed slice of the target type without making any copies.
+/// Panics if the cast is unsuccessful in debug mode, otherwise an empty slice is returned.
```

Comment:
nit: May I know why we do not panic in release? Fail to cast IMHO sounds like a big problem that should have an error

Reply:
To be replaced with bytemuck.